### PR TITLE
update explain-mpp.md: add explanation for `MppVersion` and `Compression`

### PR DESCRIPTION
**Pull request by solotzg**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: explain-mpp.md [hunk 1]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
 
 与 `EXPLAIN` 相比，ExchangeSender 的 `operator info` 中多了 `task id` 的输出，其记录了该查询片段实例化成的 MPP 任务的任务 ID。此外 MPP 算子中都会有 `threads` 这一列，这列记录了 MPP 在执行该算子时使用的并发数（如果集群由多个节点组成，该并发数是所有节点并发数相加的结果）。
 
+## MPP Version
+
+从 `v6.6.0` 开始，MPP 执行计划新增字段 `MppVersion` 和 `Compression`。
+
+- `MppVersion`：MPP 执行计划的版本号，可通过系统变量 `mpp_version` 设置。
+- `Compression`：Exchange 算子的数据压缩模式，可通过系统变量 `mpp_exchange_compression_mode` 设置，不启用数据压缩则该字段不显示。
```
Comment (joey-yez) [inline explain-mpp.md:173]:
```suggestion
- `Compression`：Exchange 算子的数据压缩模式，可通过系统变量 `mpp_exchange_compression_mode` 设置。如果未启用数据压缩，则不显示该字段。
```

File: explain-mpp.md [hunk 2]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
...
+|                 └─TableFullScan_30     | 600037902.00 | mpp[tiflash] | table:lineitem | keep order:false                                                                                                                                                                                                                                                                     |
++----------------------------------------+--------------+--------------+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+例如上面的 `EXPLAIN` 结果中，TiDB 使用了版本为 `1` 的 MPP 执行计划来构建 `TableReader`，其中类型为 `HashPartition` 的 ExchangeSender 使用 `FAST` 数据压缩模式，类型为 `PassThrough` 的 ExchangeSender 不启用数据压缩。
```
Comment (joey-yez) [inline explain-mpp.md:194]:
```suggestion
例如上面的 `EXPLAIN` 结果中，TiDB 使用了版本为 `1` 的 MPP 执行计划来构建 `TableReader`。其中类型为 `HashPartition` 的 ExchangeSender 使用 `FAST` 数据压缩模式；类型为 `PassThrough` 的 ExchangeSender 未启用数据压缩。
```

File: explain-mpp.md [hunk 3]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
 
 与 `EXPLAIN` 相比，ExchangeSender 的 `operator info` 中多了 `task id` 的输出，其记录了该查询片段实例化成的 MPP 任务的任务 ID。此外 MPP 算子中都会有 `threads` 这一列，这列记录了 MPP 在执行该算子时使用的并发数（如果集群由多个节点组成，该并发数是所有节点并发数相加的结果）。
 
+## MPP Version
```
Comment (TomShawn) [inline explain-mpp.md:168]:
```suggestion
## 压缩 TiFlash 在节点上的数据交换
```

File: explain-mpp.md [hunk 4]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
 
 与 `EXPLAIN` 相比，ExchangeSender 的 `operator info` 中多了 `task id` 的输出，其记录了该查询片段实例化成的 MPP 任务的任务 ID。此外 MPP 算子中都会有 `threads` 这一列，这列记录了 MPP 在执行该算子时使用的并发数（如果集群由多个节点组成，该并发数是所有节点并发数相加的结果）。
 
+## MPP Version & Exchange 数据压缩
```
Comment (TomShawn) [inline explain-mpp.md:168]:
```suggestion
## MPP Version 和 Exchange 数据压缩
```

File: explain-mpp.md [hunk 5]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
 
 与 `EXPLAIN` 相比，ExchangeSender 的 `operator info` 中多了 `task id` 的输出，其记录了该查询片段实例化成的 MPP 任务的任务 ID。此外 MPP 算子中都会有 `threads` 这一列，这列记录了 MPP 在执行该算子时使用的并发数（如果集群由多个节点组成，该并发数是所有节点并发数相加的结果）。
 
+## MPP Version 和 Exchange 数据压缩
+
+从 `v6.6.0` 开始，MPP 执行计划新增字段 `MppVersion` 和 `Compression`。
```
Comment (TomShawn) [inline explain-mpp.md:170]:
```suggestion
从 v6.6.0 开始，MPP 执行计划新增字段 `MppVersion` 和 `Compression`。
```

File: explain-mpp.md [hunk 6]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
...
+
+- `MppVersion`：MPP 执行计划的版本号，可通过系统变量 `mpp_version` 设置。
+- `Compression`：Exchange 算子的数据压缩模式，可通过系统变量 `mpp_exchange_compression_mode` 设置。如果未启用数据压缩，则不显示该字段。
```
Comment (TomShawn) [inline explain-mpp.md:173]:
```suggestion
- `MppVersion`：MPP 执行计划的版本号，可通过系统变量 [`mpp_version`](/system-variables.md#mpp_version-从-v660-版本开始引入) 设置。
- `Compression`：`Exchange` 算子的数据压缩模式，可通过系统变量 [`mpp_exchange_compression_mode`](/system-variables.md#mpp_exchange_compression_mode-从-v660-版本开始引入) 设置。如果未启用数据压缩，则不显示该字段。
```

File: explain-mpp.md [hunk 7]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
 
 与 `EXPLAIN` 相比，ExchangeSender 的 `operator info` 中多了 `task id` 的输出，其记录了该查询片段实例化成的 MPP 任务的任务 ID。此外 MPP 算子中都会有 `threads` 这一列，这列记录了 MPP 在执行该算子时使用的并发数（如果集群由多个节点组成，该并发数是所有节点并发数相加的结果）。
 
+## MPP Version 和 Exchange 数据压缩
```
Comment (TomShawn) [inline explain-mpp.md:168]: 这一部分能否说明一下需要数据压缩的使用场景，对用户有什么好处吗？

File: explain-mpp.md [hunk 8]
```diff
@@ -165,6 +165,34 @@ EXPLAIN ANALYZE SELECT COUNT(*) FROM t1 GROUP BY id;
...
+|                 └─TableFullScan_30     | 600037902.00 | mpp[tiflash] | table:lineitem | keep order:false                                                                                                                                                                                                                                                                     |
++----------------------------------------+--------------+--------------+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+例如上面的 `EXPLAIN` 结果中，TiDB 使用了版本为 `1` 的 MPP 执行计划来构建 `TableReader`。其中类型为 `HashPartition` 的 ExchangeSender 使用 `FAST` 数据压缩模式；类型为 `PassThrough` 的 ExchangeSender 未启用数据压缩。
```
Comment (TomShawn) [inline explain-mpp.md:194]:
```suggestion
例如上面的执行计划结果中，TiDB 使用了版本为 `1` 的 MPP 执行计划来构建 `TableReader`。其中类型为 `HashPartition` 的 ExchangeSender 算子使用 `FAST` 数据压缩模式，类型为 `PassThrough` 的 ExchangeSender 算子未启用数据压缩。
```

**Comment by solotzg** [comment]
Use #12922

**Pull request closed without merging**